feature_gates/__init__.py

```diff
@@ -0,0 +1,136 @@
+from dataclasses import dataclass
+from configparser import ConfigParser, NoOptionError, NoSectionError
+from typing import Final
+from pathlib import Path
+
+
+@dataclass
```

Comment:
If folks find this objectionable, this can be seperated out into a seperate class file, rather than doing everything in the `__init__.py`